unit/java_bytecode/java_types/erase_type_arguments.cpp

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************\
+
+ Module: Unit tests for java_types
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+#include <testing-utils/catch.hpp>
+#include <java_types.h>
+
+SCENARIO("erase_type_arguments", "[core][java_types]")
+{
+  THEN(
+    "erase_type_arguments should leave strings with no type arguments "
+    "unaltered.")
+  {
+    const std::string testInput1 = "testString1";
```

Comment:
Nit - no need for a variable, give it as an input to the function directly as in the rest of the file.